public/app/features/explore/ExploreQueryInspector.test.tsx

```diff
@@ -33,6 +34,15 @@ jest.mock('@grafana/runtime', () => ({
   reportInteraction: () => null,
 }));
 
+jest.mock('react-virtualized-auto-sizer', () => {
+  return {
+    __esModule: true,
+    default(props: any) {
+      return <div>{props.children({ height: 1000 })}</div>;
+    },
+  };
+});
+
```

Comment:
```suggestion
jest.mock('react-virtualized-auto-sizer', () => {
  return {
    __esModule: true,
    default(props: ComponentProps<typeof AutoSizer>) {
      return <div>{props.children({ width: 1000, height: 1000 })}</div>;
    },
  };
});
```

to avoid `any`